### ISSUE TITLE
Gives an error / warning in php 5.4 in strict mode

### ISSUE THREAD

**Issue by andrearennick** [copyblogger/genesis-tabs#7]
Customer reporting:

"I installed one of your plugins:
Genesis Tabs v 0.9.3

I get a the following php warning:

Strict Standards: call_user_func_array() expects parameter 1 to be a valid callback, non-static method Genesis_Tabs::init() should not be called statically in ... on line 496

I solved it by changing the following in plugin.php:

line 40:
$tabs = new Genesis_Tabs();
add_action( 'after_setup_theme', array( $tabs, 'init' ) );

line 55 to 60:
add_action( 'widgets_init', array( $this, 'register_widget' ) );

add_action( 'wp_print_styles', array( $this, 'register_styles' ) );
add_action( 'wp_enqueue_scripts', array( $this, 'register_scripts' ) );

add_action( 'wp_footer', array( $this, 'footer_js' ), 20 );

To summarize, I initialize the class and I pass the class as parameter to the add_action.

Using PHP 5.4."
